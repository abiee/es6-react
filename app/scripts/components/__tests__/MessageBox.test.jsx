jest.dontMock('../MessageBox.jsx');

import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import MessageBox from '../MessageBox.jsx';

describe('HelloWorld component', () => {
  it('should display the right message', () => {
    const tree = renderer.create(
      <MessageBox title="Example title" message="Hello world!" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
