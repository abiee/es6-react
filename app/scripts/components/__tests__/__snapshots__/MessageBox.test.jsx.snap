exports[`HelloWorld component should display the right message 1`] = `
<div>
  <h4>
    Example title
  </h4>
  <p>
    Hello world!
  </p>
</div>
`;
